src/redismodule.rs

```diff
@@ -31,31 +32,43 @@ impl<T> NextArg for T
 where
     T: Iterator<Item = RedisString>,
 {
+    #[inline]
     fn next_arg(&mut self) -> Result<RedisString, RedisError> {
         self.next().map_or(Err(RedisError::WrongArity), Ok)
     }
 
+    #[inline]
     fn next_string(&mut self) -> Result<String, RedisError> {
         self.next()
             .map_or(Err(RedisError::WrongArity), |v| Ok(v.into_string_lossy()))
     }
 
+    #[inline]
+    fn next_str<'a>(&mut self) -> Result<&'a str, RedisError> {
+        self.next()
+            .map_or(Err(RedisError::WrongArity), |v| v.try_as_str())
```

Comment:
`next_string` is doing a lossy conversion (calling indirectly to `String::from_utf8_lossy`)
And `next_str` is not (calling indirectly to `str::from_utf8`)
Is this intended?

Reply:
yes, I think we should reconsider the from_utf8_lossy